neuralmonkey/learning_utils.py

```diff
@@ -520,21 +565,21 @@ def _log_model_variables() -> None:
     trainable_vars = tf.trainable_variables()
     total_params = 0
 
-    logstr = "The model has {} trainable variables:\n\n".format(
-        len(trainable_vars))
+    logstr = "The model has {} trainable variables:\n\n"\
```

Comment:
Tentokrat s tebou souhlasim a opravim (taky se mi to nelibi), ale opet bych se ohradil k tomu, ze si podle me vymejslis ... celkove kod opice nema moc jednotny styl (viz konverzace vyse) a zkusil jsem si to lomitko na konci radku v opici najit a je minimalne 10x